<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDlgEditElectricWpt</class>
 <widget class="QDialog" name="IDlgEditElectricWpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Electric Data of Waypoint ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizeIncrement">
           <size>
            <width>2</width>
            <height>0</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Data</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>11</y>
              <width>405</width>
              <height>198</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Consumers (total)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinConsumers">
                 <property name="locale">
                  <locale language="English" country="UnitedStates"/>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Phase 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinConsumers1">
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinConsumers2">
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinConsumers3">
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Load (total)</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QSpinBox" name="spinLoad">
                   <property name="maximum">
                    <number>100000</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Watt</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Phase 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinLoad1">
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinLoad2">
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinLoad3">
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="powerFactorLabel">
                 <property name="text">
                  <string>Power factor</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinPowerfactor">
                 <property name="minimum">
                  <double>0.200000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.750000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>More</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWebView" name="lineMore">
              <property name="url">
               <url>
                <string>about:blank</string>
               </url>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IDlgEditElectricWpt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IDlgEditElectricWpt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
